### DISCUSSION TITLE
academy/api-scraping/general-api-scraping/locating-and-learning

### DISCUSSION THREAD

**Discussion by giscus** [apify/apify-docs#1149] [Comments]
# academy/api-scraping/general-api-scraping/locating-and-learning

Learn how to effectively locate a website's API endpoints, and learn how to use them to get the data you want faster and more reliably.

https://docs.apify.com/academy/api-scraping/general-api-scraping/locating-and-learning

**Comment by howardkrauth** [comment]
I stumbled upon this while trying to figure out the API for a website.  The section on locating API endpoints for a site with no known API is the best answer I've ever found on the internet for anything.  Thank you!
